code/week2/operations.cpp

```diff
@@ -0,0 +1,25 @@
+int add(int a, int b) {
+    return a + b;
+}
+
+int subtract(int a, int b) {
+    return add(a, -b);
+}
+
+int multiply(int a, int b) {
+    int accum = 0;
+    for(int i = 0; i < a; i++) {
```

Comment:
I like the chaining of functions in the example (also example code shouldn't reeeeeeally always be optimized for the compiler)